### ISSUE TITLE
LTE_Tr (and LTE_Te) does not calcuate ionization for macro atoms in LTE

### ISSUE THREAD

**Issue by kslong** [sirocco-rt/sirocco#1103]
LTE is used to initialize a Python cycle, but in LTE mode even in subsequent cycles macro atoms are ionized based on estimators due to photon passages. As a result, macro atoms are not held in LTE for the LTE_Tr and LTE_te modes.  This should not be the case.  The LTE modes should force all ions, macro or otherwise, to LTE based on Tr or Te.

(Note that it is unclear to me how some of the other "simple" modes operate)>

**Comment by jhmatthews** [comment]
Knox, is this closed by the referenced PR above? #1173 ?

**Comment by kslong** [comment]
Yes, @jhmatthews  This should be resolved now, so I am closing the issue.
